src/Aether/Devices/Drivers/Sgp4x.cs

```diff
@@ -0,0 +1,174 @@
+
+            // Read serial number into array excluding CRC bytes
+            byte[] serialNumber = new byte[6];
+
+            serialNumber[0] = serialNumberWithCRC[0];
+            serialNumber[1] = serialNumberWithCRC[1];
+            serialNumber[2] = serialNumberWithCRC[3];
+            serialNumber[3] = serialNumberWithCRC[4];
+            serialNumber[4] = serialNumberWithCRC[6];
+            serialNumber[5] = serialNumberWithCRC[7];
+
+            return new ReadOnlySpan<byte>(serialNumber);
+        }
+
+        /// <summary>
+        /// Runs a self test on the device.
+        /// </summary>
+        /// <returns>True if all tests passed. False if one or more tests failed.</returns>
+        public bool RunSelfTest()
+        {
+            Span<byte> runSelfTestCommand = stackalloc byte[2] { 0x28, 0x0E };
+            Span<byte> testResultWithCRC = stackalloc byte[3];
+
+            // Write run self test command
+            _device.Write(runSelfTestCommand);
+
+            Thread.Sleep(320);
+
+            // Read test result data
+            _device.Read(testResultWithCRC);
+
+            Sensirion.ReadUInt16BigEndianAndCRC8(testResultWithCRC);
+
+            // Check result status (Most significant byte, ignore least significant non-crc byte)
+            // 0xD4 = All tests passed
+            // 0x4B = One or more test failed
+            return testResultWithCRC[0] == 0xD4;
+        }
+
+        /// <summary>
+        /// Gets the raw VOC measurements from the device.
+        /// </summary>
+        /// <param name="relativeHumidityValue">The relative humidity value expressed as a percentage.</param>
+        /// <param name="temperatureValue">The temperature value expressed in degrees celsius.</param>
+        /// <returns>The raw VOC measurement value from the sensor. If an error occurred, it will be <see langword="null"/>.</returns>
+        /// <remarks>If default relative humidity and temperature values are supplied, humidity compensation will be disabled.</remarks>
+        public VolumeConcentration? GetVOCRawMeasure(ushort relativeHumidityValue = 50, short temperatureValue = 25)
+        {
+            if (relativeHumidityValue > 100)
+            {
+                throw new ArgumentOutOfRangeException(nameof(relativeHumidityValue), relativeHumidityValue, "A relative humidity percentage value must be between 0 and 100");
+            }
+
+            if (temperatureValue < -45 || temperatureValue > 130)
+            {
+                throw new ArgumentOutOfRangeException(nameof(temperatureValue), temperatureValue, "A temperature value must be between -45 and 130");
+            }
+
+            Span<byte> writeBuffer = stackalloc byte[8];
+            Span<byte> readBuffer = stackalloc byte[3];
+
+            // Write start measure VOC raw command
+            writeBuffer[0] = 0x26;
+            writeBuffer[1] = 0x0F;
+
+            // Write default humidity value + CRC (0x80, 0x00, [CRC], 0xA2)
```

Comment:
What is the `0xA2`? This method only writes 3 bytes. I don't think this is a default value anymore either :).